src/main/java/bg/libapp/libraryapp/model/mappers/AuthorMapper.java

```diff
@@ -0,0 +1,51 @@
+package bg.libapp.libraryapp.model.mappers;
+
+import bg.libapp.libraryapp.model.dto.author.AuthorBookViewDTO;
+import bg.libapp.libraryapp.model.dto.author.AuthorViewDTO;
+import bg.libapp.libraryapp.model.entity.Author;
+import bg.libapp.libraryapp.repository.AuthorRepository;
+import bg.libapp.libraryapp.repository.BookRepository;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Lazy;
+import org.springframework.stereotype.Component;
+
+import java.util.stream.Collectors;
+
+@Component
+public class AuthorMapper {
```

Comment:
Refactored as shown below:
`mapBookAuthorViewDTOFromBook -> toBookAuthorViewDTO`
`mapBookFromBookAddDTO -> toBook`
`mapBookViewDTOFromBook -> toBookViewDTO`
and etc. Thats the right way?

Reply:
Without "View" as mentioned in one of the above comments.

Reply:
I think that the name of the method should contain some verb, so I think it will be better to be 'convertTo... ', for example, although it is clear that it is a mapper.